src/test/CourseNavigation.test.jsx

```
@@ -0,0 +1,113 @@
+    // Should show lessons after clicking
+    expect(screen.getByText(/Lessons/)).toBeInTheDocument();
+    expect(screen.getByText("Greetings and Introductions")).toBeInTheDocument();
+  });
+
+  test("displays overall progress summary", () => {
+    renderWithRouter(<CourseNavigation />);
+
+    expect(screen.getByText("Overall Progress")).toBeInTheDocument();
+  });
+
+  test("shows lesson completion status", () => {
```

ESLint

[error] 'test' is not defined.

(no-undef)